tests/TestCase.php

```diff
@@ -4,17 +4,26 @@
 
 namespace Yiisoft\Form\Tests;
 
-use Yiisoft\Form\Tests\Stubs\LoginForm;
+use Yiisoft\Composer\Config\Builder;
+use Yiisoft\Di\Container;
+use Yiisoft\Form\Tests\Stub\LoginForm;
+use Yiisoft\Widget\WidgetFactory;
 
 class TestCase extends \PHPUnit\Framework\TestCase
 {
+    private Container $container;
     protected LoginForm $loginForm;
 
     protected function setUp(): void
     {
         parent::setUp();
 
+        $config = require Builder::path('tests');
```

Comment:
That could be removed if we'll remove events, right?

Reply:
done.